openfecwebapp/templates/partials/legal-search-results-statute.html

```diff
@@ -1,18 +1,14 @@
 <div class="simple-table simple-table--responsive simple-table--display legal-search-results">
```

Comment:
@adborden couldn't find `legal-search-results` class in fec-style.  Not sure what this was for.